requirements.txt

```diff
@@ -1,2 +1,4 @@
 beautifulsoup4==4.11.1
+lxml==4.9.1
+requests==2.28.1
 selenium==4.2.0
```

Comment:
now that we've gotten rid of old scripts, i dont think we need selenium anymore

Reply:
Cool, tested it without selenium and update.py works fine.